tests/OmniSharp.Tests/BufferManagerFacts.cs

```diff
@@ -10,7 +9,7 @@ namespace OmniSharp.Tests
     public class BufferManagerFacts
     {
         [Fact]
-        public async Task UpdateBufferIgnoresVoidRequests()
+        public void UpdateBufferIgnoresVoidRequests()
```

Comment:
it works for me in beta4 using the latest mono CI build, just gotta wait for a nice new shiny mono release.